### PR TITLE
Review and update README instructions

### PR DESCRIPTION
**Pull request by ktsuench**
Reading and following along with the instructions of the README to see if there are things to expand on or adjusta.

Also found another small change in .asf.yaml, spelling mistake with the field `required_approving_review_count`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,16 +61,28 @@ This extension contributes the following commands to the Command palette.
...
 
-```bash
-git clone https://github.com/apache/openwhisk-vscode-extension.git
-code ./openwhisk-vscode-extension # Open openwhisk-vscode-extension in VSCode
-```
+### Requirements
+
+- [vscode](https://code.visualstudio.com/) >= 1.41.0
+- [Node.js](https://nodejs.org/en/download/) >= 12.x
+- Have a OpenWhisk deployment available, can get set up by following [guide](https://openwhisk.apache.org/documentation.html#pre-requisites)
```
Comment (rabbah) [inline README.md:70]:
```suggestion
- Have an OpenWhisk deployment available, which you set up by following [guide](https://openwhisk.apache.org/documentation.html#pre-requisites) or by using one of the managed OpenWhisk offerings (e.g., from IBM, Adobe or Nimbella).
```

File: README.md [hunk 2]
```diff
@@ -61,16 +61,28 @@ This extension contributes the following commands to the Command palette.
...
+1. Clone the repository:
 
-Press `F5` button(run debug)
+    ```bash
+    git clone https://github.com/apache/openwhisk-vscode-extension.git
+    code ./openwhisk-vscode-extension # Open openwhisk-vscode-extension in VS Code
+    ```
 
-Learn more about debugging [here](https://github.com/apache/openwhisk-vscode-extension/blob/master/vsc-extension-quickstart.md)
+2. In terminal execute command `npm install` to install extension dependencies
```
Comment (rabbah) [inline README.md:81]:
```suggestion
2. In your terminal, execute the command `npm install` to install extension dependencies
```

File: README.md [hunk 3]
```diff
@@ -61,16 +61,28 @@ This extension contributes the following commands to the Command palette.
...
+3. In terminal execute command `npm run webpack-dev` to start up the extension builder in watch mode (allows for making changes to files and rebuilding automatically)
```
Comment (rabbah) [inline README.md:82]:
```suggestion
3. In your terminal, execute the command `npm run webpack-dev` to start up the extension builder in watch mode (allows for making changes to files and rebuilding automatically)
```

**Comment by rabbah** [comment]
Thanks for the contribution @ktsuench 🎉

File: README.md (hunk 1 above)
Comment (ktsuench) [reply README.md]: Added this in commit be186d8

File: README.md (hunk 2 above)
Comment (ktsuench) [reply README.md]: Added this in commit be186d8

File: README.md (hunk 3 above)
Comment (ktsuench) [reply README.md]: Added this in commit be186d8

**Comment by style95** [comment]
Hope this is merged after the first release.

**Comment by rabbah** [comment]
@style95 the PR can be merged ahead of the actual release since the release itself is specific to a commit and the release tag is also specific to that commit.

Do you have a concern about merging this sooner?

**Comment by style95** [comment]
I am ok with merging it in paralle.

**Review by style95 — APPROVED** [review]

**Pull request merged**